Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -8513,15 +8505,15 @@ void WebPage::urlSchemeTaskDidComplete(WebURLSchemeHandlerIdentifier handlerIden
     handler->taskDidComplete(taskIdentifier, error);
 }
 
-void WebPage::setIsSuspended(bool suspended)
+void WebPage::setIsSuspended(bool suspended, CompletionHandler<void(std::optional<bool>)>&& completionHandler)
 {
     if (m_isSuspended == suspended)
-        return;
+        return completionHandler({ });
```

Comment:
Is this the preferred style in WebKit now that we have so many completion handler related braces? `return completionHandler()` seems like an oxymoron/typo at first read and might be an error in some languages.

Reply:
I'm not aware of a preferred style. I see both used often.